packages/frontend/components/Borrow/PreviewTransaction.tsx

```diff
@@ -0,0 +1,311 @@
+import React, { useState } from "react"
+import { useTheme } from "@mui/material/styles"
+import {
+  Box,
+  ButtonBase,
+  Card,
+  CardActionArea,
+  CardContent,
+  Collapse,
+  Container,
+  Dialog,
+  DialogContent,
+  Divider,
+  Grid,
+  Tooltip,
+  Typography,
+} from "@mui/material"
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined"
+import AddCircleIcon from "@mui/icons-material/AddCircle"
+import CancelIcon from "@mui/icons-material/Cancel"
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore"
+import Image from "next/image"
+
+import LTVProgressBar from "./LTVProgressBar"
+import ClickableTooltip from "../Layout/ClickableTooltip"
+
+export default function PreviewTransaction() {
```

Comment:
General thought with this component: I think we overcomplicated it with a lot of nested `Grid` (your bff haha).

This is something we should discuss directly cause it's difficult to explain by message 🤔

Reply:
Unless I'm mistaken, `<Grid item>` doesn't accept `justify-content` prop.
To fix it, I replaced it by a `sx={{ display: "flex", justify-content: "center" }}`. Not sure to be a better way...